packages/api-clients/open-api/tenantApi.yml

```diff
@@ -778,6 +778,30 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /maintenance/tenants/{tenantId}:
```

Comment:
for maintenance routes I suggest the use of the additional (body) parameter `version`.
The idea is to manually pass the current version of the object we are updating/deleting, to be sure that we are not making changes on a stale object.

cc @beetlecrunch

Reply:
It's a good idea.
Ok, let's introduce a dedicated payload containing the version of the object we are managing.

Reply:
I've done a bit of refactoring, but the main changes are [here](https://github.com/pagopa/interop-be-monorepo/pull/837/commits/0d605211b2b347527fed962174411e30a5ff6c23)

Reply:
thanks!
by re-reading the spec I noticed that future us may be confused by which value to use (the existing one? the next one?).
May I ask you to rename it to `currentVersion`?
sorry for the rework